coba/pipes/rows.py

```diff
@@ -513,7 +513,13 @@ def filter(self, rows: Iterable[Union[Any,Dense,Sparse]]) -> Iterable[Union[Any,
             rows = rows
 
         if self._tipe =='onehot':
-            rows = Flatten().filter(rows)
+            if isinstance(first, Dense) and hasattr(first, "headers"): # do all dense rows have headers?
```

Comment:
maybe only openml datasets have .headers

Reply:
Yeah, I didn't want to assume all dense rows have headers. 

For example, in unittests I often just pass in either a list of lists or a list of dicts as rows.

All openml datasets will because the arff file format does.